common/segment_error.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+parent: Common
+---
+
+# Segment Error
+
+Understanding the `SegmentError` exception.
+{: .fs-6 .fw-300 }
+
+&nbsp;
+
+{: .warning }
+> _A **Segment** unknown to pyracf would generate an improper **EXTRACT** request to IRRSMO00. Pyracf will always raise a `SegmentError` to bring attention to any unknown values prior to executing such a request._
```

Comment:
I don't think the warning annotation is necessary here.

Reply:
Fine by me, my intention was consistency with the existing error types. This warning has been removed.